pkg/web/routes.go

```diff
@@ -123,12 +130,33 @@ func (wa *WebAPI) Router() *mux.Router {
 // route handlers
 // ==============
 
+func (wa *WebAPI) logeventsSave(w http.ResponseWriter, r *http.Request) {
```

Comment:
Are we only exposing this endpoint for testing purposes? If not, what's the value in exposing an endpoint that allows you to write arbitrary events?

Reply:
It was for testing, deleted now in final code